StRoot/StEvent/StETofHeader.cxx

```diff
@@ -55,13 +59,15 @@ StETofHeader::StETofHeader( const double& trgGdpbTime, const double& trgStarTime
     setRocGdpbTs( gdpbTs );
     setRocStarTs( starTs );
     const size_t kNbGet4sInSystem = 1728;
-    mMissMatchFlagVec = vector<bool>( kNbGet4sInSystem, false ); 
+    mMissMatchFlagVec = std::vector<bool>( kNbGet4sInSystem, false );
+    const size_t kNbCountersInSystem = 108;
+    mGoodEventFlagVec = std::vector<bool>( kNbCountersInSystem, false );  
```

Comment:
You should use the initializer list to initialize the new data members in all constructors.